modules/home-manager/lsd.nix

```
@@ -0,0 +1,20 @@
+{ catppuccinLib }:
+{ config, lib, ... }:
+
+let
+  inherit (config.catppuccin) sources;
+
+  cfg = config.catppuccin.lsd;
+  enable = cfg.enable && config.programs.lsd.enable;
+in
+
+{
+  options.catppuccin.lsd = catppuccinLib.mkCatppuccinOption { name = "lsd"; };
+
+  config = lib.mkIf enable {
+    xdg.configFile = {
+      "lsd/config.yaml".source = "${sources.lsd}/catppuccin-${cfg.flavor}/colors.yaml";
```

Just noticed this should be `colors.yaml` not `config.yaml`
```suggestion
      "lsd/colors.yaml".source = "${sources.lsd}/catppuccin-${cfg.flavor}/colors.yaml";
```